package.json

```diff
@@ -7,66 +7,64 @@
     "foundation-sites": "^5.5.3",
     "jquery": "^3.3.1",
-    "jquery-migrate": "^1.4.1",
     "jstree": "vakata/jstree",
     "lazysizes": "4.1.2",
     "lodash": "^4.17.4",
     "nod-validate": "^2.0.12",
     "pace": "hubspot/pace#a03f1f1de62c9cea6c88b2267b8d7a83858b6cb6",
     "slick-carousel": "^1.8.1",
     "svg-injector": "^1.1.3",
-    "sweetalert2": "^6.10.1"
+    "sweetalert2": "^6.11.5"
   },
   "devDependencies": {
+    "@babel/core": "^7.2.0",
+    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
+    "@babel/preset-env": "^7.2.0",
     "@bigcommerce/citadel": "^2.15.1",
-    "babel-core": "^6.26.0",
-    "babel-eslint": "^8.2.2",
-    "babel-loader": "^7.1.4",
-    "babel-plugin-dynamic-import-webpack": "^1.0.2",
-    "babel-plugin-lodash": "^3.3.2",
-    "babel-plugin-syntax-dynamic-import": "^6.18.0",
-    "babel-plugin-transform-regenerator": "^6.26.0",
-    "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-preset-env": "^1.6.1",
+    "@bigcommerce/stencil-cli": "^1.15.4",
+    "babel-loader": "^8.0.4",
+    "babel-plugin-dynamic-import-webpack": "^1.1.0",
+    "babel-plugin-lodash": "^3.3.4",
     "clean-webpack-plugin": "^0.1.19",
-    "core-js": "^2.5.0",
-    "es6-shim": "^0.35.3",
     "eslint": "^4.8.0",
     "eslint-config-airbnb": "^16.0.0",
     "eslint-plugin-import": "^2.7.0",
     "eslint-plugin-jsx-a11y": "^6.0.2",
     "eslint-plugin-react": "^7.4.0",
     "grunt": "^1.0.1",
+    "grunt-cli": "^1.3.2",
     "grunt-eslint": "^20.0.0",
     "grunt-karma": "^2.0.0",
     "grunt-scss-lint": "^0.5.0",
     "grunt-svgstore": "^1.0.0",
     "imports-loader": "^0.7.1",
     "jasmine-core": "^2.2.0",
     "karma": "^1.7.0",
-    "karma-babel-preprocessor": "^7.0.0",
+    "karma-babel-preprocessor": "^8.0.0-beta.0",
     "karma-coverage": "^1.1.1",
     "karma-es6-shim": "^1.0.0",
     "karma-jasmine": "^1.1.0",
     "karma-phantomjs-launcher": "^1.0.4",
     "karma-sourcemap-loader": "0.3.7",
     "karma-verbose-reporter": "0.0.6",
     "karma-webpack": "^2.0.4",
+    "lighthouse": "^4.0.0-alpha.2-3.2.1",
     "load-grunt-config": "^0.19.2",
     "lodash-webpack-plugin": "^0.11.2",
-    "regenerator-runtime": "^0.11.0",
+    "npx": "^10.2.0",
     "time-grunt": "^1.2.2",
-    "webpack": "^4.1.1",
-    "webpack-cli": "^2.0.11",
+    "webpack": "^4.27.1",
+    "webpack-bundle-analyzer": "^3.0.3",
+    "webpack-cli": "^3.1.2",
     "webpack-merge": "^4.1.2"
   },
   "scripts": {
-    "build": "webpack --config webpack.prod.js",
-    "buildDev": "webpack --config webpack.dev.js"
+    "build": "npx webpack --config webpack.prod.js",
+    "buildDev": "npx webpack --config webpack.dev.js",
+    "lighthouse": "npx lighthouse --output json --chrome-flags=\"--headless\" --quiet --disable-device-emulation --throttling-method=provided $URL | jq '{ttfb: .audits | .[\"time-to-first-byte\"] | .rawValue, paint: .audits | .[\"first-meaningful-paint\"] | .rawValue, interactive: .audits.interactive.rawValue}'"
```

Comment:
It would be pretty cool if we could surface this information on the PR as a comment.